clients/stellar-relay-lib/src/connection/connector/message_handler.rs

```diff
@@ -65,8 +65,9 @@ impl Connector {
 		&mut self,
 		msg: StellarMessage,
 		msg_type: MessageType,
+		data_len: usize,
 	) -> Result<Option<StellarMessage>, Error> {
-		match msg {
+		match msg.clone() {
```

Comment:
why clone?

Reply:
Another remnant of previous changes. Removing. Good catch!